homeassistant/components/dlna_dms/__init__.py

```diff
@@ -26,3 +27,21 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     # Forward unload to this domain's data manager
     return await get_domain_data(hass).async_unload_entry(entry)
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
If we can do a soft migration without increasing the config entry version we prefer that, as it allows roll backs. We could instead check if the source id key is missing in the config entry data, and update it if needed, during config entry setup.

Reply:
Will do.